src/FilterList.js

```diff
@@ -162,11 +162,9 @@ let FilterList = _.flow(
   }) => {
     let updateRequired =
       tree.disableAutoUpdate &&
-      // find if any nodes in the tree are marked for update (i.e. usually nodes are marked for update because they react to "others" reactor)
-      _.some(
-        treeNode => treeNode !== node && treeNode.markedForUpdate,
-        F.treeToArray(_.get('children'))(node)
-      )
+      // find if any nodes in the tree are marked for update
+      // i.e. usually nodes are marked for update because they react to "others" reactor
+      _.some('markedForUpdate', tree.flat)
```

Comment:
Let's make sure we reference this in https://github.com/smartprocure/contexture-client/issues/131

Also, technically this might not be right because it won't get subscriptions to nodes that are added afterwards (but we can revisit that later)